src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configurations>netstandard2.0-Debug;netstandard2.0-Release</Configurations>
+    <TargetFrameworks>netstandard2.0;_$(NetFrameworkCurrent)</TargetFrameworks>
+    <ExcludeTargetFrameworksFromPackages>_$(NetFrameworkCurrent)</ExcludeTargetFrameworksFromPackages>
```

Comment:
You shouldn't need to do this.  Outer build should always exclude placeholder configurations.